src/FileConventions/Library.fs

```diff
@@ -49,6 +49,24 @@ let DetectUnpinnedVersionsInGitHubCI(fileInfo: FileInfo) =
 
     latestTagInRunsOnRegex.IsMatch fileText
 
+let DetectUnpinnedDotnetToolInstallVersions(fileInfo: FileInfo) =
+    assert (fileInfo.FullName.EndsWith(".yml"))
+
+    let fileLines = File.ReadLines fileInfo.FullName
+
+    let dotnetToolInstallRegex =
+        Regex("dotnet\\s+tool\\s+install\\s+", RegexOptions.Compiled)
+
+    let unpinnedDotnetToolInstallVersions =
+        fileLines
+        |> Seq.filter(fun line -> dotnetToolInstallRegex.IsMatch line)
+        |> Seq.filter(fun line ->
+            not(line.Contains("--version")) && not(line.Contains("-v"))
```

Comment:
@realmarv line can have less parentheses (in particular, 4 chars less)

Reply:
https://github.com/nblockchain/conventions/pull/110